### DISCUSSION TITLE
What is the official URL and who is the official publisher of the cloud code intellij plugin?

### DISCUSSION THREAD

**Discussion by btanev** [GoogleCloudPlatform/cloud-code-intellij#3182] [Q&A] [answered]
In the [official installation instructions:](https://cloud.google.com/code/docs/intellij/install#installing) there is no link to plugins.jetbrains.com so I'm not sure what exactly is the official intellij plugin for google cloud.

Searching on the marketplace I found the following plugin that seem to be the one referenced in the documentation.
<img width="748" alt="Screenshot 2024-02-10 at 14 46 45" src="https://github.com/GoogleCloudPlatform/cloud-code-intellij/assets/1431480/4099d817-33c3-4641-82f9-99cba96d5f8b">
But the publisher is not "Google" and I don't feel comfortable installing it and giving it my GCP credentials.

For example in the [VS Code documentation](https://cloud.google.com/code/docs/vscode/install) there is a link to the [marketplace](https://marketplace.visualstudio.com/items?itemName=GoogleCloudTools.cloudcode&ssr=false#overview) and the publisher is verified as "Google Cloud".

The question is: What is the link to the official (supported by google) intellij google cloud plugin?

**Comment by ivanporty** [comment, marked as answer]
We are in the process moving to the single Google organization as required by new Jetbrains markeplace requirements, and temporarily made one of us (me) the vendor to ublock the update - this is the correct plugin and vendor should update to Google soon. The link is here https://plugins.jetbrains.com/plugin/8079-google-cloud-code. Sorry about the inconvenience!

**Comment by emmanuelbaah** [comment]
The vendor update has been completed. Again, apologies for the inconvenience!

![image](https://github.com/GoogleCloudPlatform/cloud-code-intellij/assets/16374713/b63d99a0-f31d-47bb-aa10-dc234f86cd42)

**Comment by ivanporty** [comment]
Closing. Please reopen if something is not updated for you.
